cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/distributed/TransactionServiceTest.java

```diff
@@ -202,6 +210,12 @@ static TransactionService createTxService(String zkConnectionString, int txServi
                            new ZKClientModule(),
                            new DiscoveryRuntimeModule().getDistributedModules(),
                            new TransactionMetricsModule(),
+                           new AbstractModule() {
+                             @Override
+                             protected void configure() {
+                               bind(NamespaceQueryAdmin.class).to(SimpleNamespaceQueryAdmin.class);
```

Comment:
do you think it makes sense to extract a public ``AbstractModule`` for this binding in cdap-data-fabric/src/test/java/.../data2 and use it wherever you need it. Basically looking to see if you can skip using Modules.override altogether.